app/src/org/commcare/tasks/UpdateTask.java

```diff
@@ -236,6 +238,10 @@ public void setAsAutoUpdate() {
         wasTriggeredByAutoUpdate = true;
     }
 
+    public void setAsLocalUpdate() {
+        isLocalUpdate = true;
+    }
```

Comment:
Instead of having this setter, can you set `isLocalUpdate` in the constructor based on the value of `CommCareApplication._().isConsumerApp()` (and also make `isLocalUpdate` be a `final` field)